src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -49,8 +48,8 @@ type UserConfig interface {
 
 // NewFetcher creates a new configuration fetcher with the specified
 // interface for obtaining the user ID and token if those are populated.
-func NewFetcher(conf UserConfig, httpFetcher util.HTTPFetcher) Fetcher {
-	return &fetcher{lastCloudConfigETag: map[string]string{}, user: conf, httpFetcher: httpFetcher}
+func NewFetcher(conf UserConfig, httpClient *http.Client) Fetcher {
```

Comment:
I dig it, it's just that previously we had some methods that returned an HTTPFetcher and some that returned an *http.Client.  http.Client is actually a superset of HTTPFetcher, and I value consistency, so I went with returning http.Client across the board.  What do you think @myleshorton ?